README.md

```diff
@@ -7,7 +7,7 @@
 
 **Requires at least:** 3.0
 
-**Tested up to:** 5.3.2
+**Tested up to:** 5.4
```

Comment:
@umeshnevase Also, update in readme.txt and main plugin file.

Reply:
@chandrapatel okay